<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CropDialog</class>
 <widget class="QDialog" name="CropDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crop</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Controls</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,1,1,1,0,0,0">
       <item row="3" column="2">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxXMax"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelX">
         <property name="text">
          <string>x:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelZ">
         <property name="text">
          <string>z:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labelMin">
         <property name="text">
          <string>Min</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="labelMax">
         <property name="text">
          <string>Max</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxYMax"/>
       </item>
       <item row="5" column="3" colspan="2">
        <widget class="QLineEdit" name="lineEditZDelta"/>
       </item>
       <item row="7" column="2">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="6">
        <widget class="QCheckBox" name="checkBoxXSync">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="5">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxYMin"/>
       </item>
       <item row="5" column="6">
        <widget class="QCheckBox" name="checkBoxZSync">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxXMin"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelY">
         <property name="text">
          <string>y:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="3" colspan="2">
        <widget class="QLineEdit" name="lineEditXDelta"/>
       </item>
       <item row="4" column="6">
        <widget class="QCheckBox" name="checkBoxYSync">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="3" colspan="2">
        <widget class="QLineEdit" name="lineEditYDelta"/>
       </item>
       <item row="6" column="4">
        <widget class="QSpinBox" name="spinBoxCropAngle"/>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxZMin"/>
       </item>
       <item row="1" column="3" colspan="2">
        <widget class="QLabel" name="labelDelta">
         <property name="text">
          <string>Delta</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="2" colspan="2">
        <widget class="QLabel" name="labelCropAngle">
         <property name="text">
          <string>Crop Angle (deg):</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxZMax"/>
       </item>
       <item row="3" column="7">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="5" rowspan="2" colspan="3">
        <widget class="QLabel" name="labelSync">
         <property name="text">
          <string>Max=Min</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Cropped-Away Pieces</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="labelRedHeading">
         <property name="text">
          <string>Red:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labelRed">
         <property name="text">
          <string>0.00</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSlider" name="sliderRed">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelGreenHeading">
         <property name="text">
          <string>Green:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelGreen">
         <property name="text">
          <string>0.00</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSlider" name="sliderGreen">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelBlueHeading">
         <property name="text">
          <string>Blue:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="labelBlue">
         <property name="text">
          <string>0.00</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QSlider" name="sliderBlue">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelOpacityHeading">
         <property name="text">
          <string>Opacity:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="labelOpacity">
         <property name="text">
          <string>0.00</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QSlider" name="sliderOpacity">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="checkBoxShow">
         <property name="text">
          <string>Show</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDefault">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDone">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>doubleSpinBoxXMin</tabstop>
  <tabstop>doubleSpinBoxXMax</tabstop>
  <tabstop>lineEditXDelta</tabstop>
  <tabstop>checkBoxXSync</tabstop>
  <tabstop>doubleSpinBoxYMin</tabstop>
  <tabstop>doubleSpinBoxYMax</tabstop>
  <tabstop>lineEditYDelta</tabstop>
  <tabstop>checkBoxYSync</tabstop>
  <tabstop>doubleSpinBoxZMin</tabstop>
  <tabstop>doubleSpinBoxZMax</tabstop>
  <tabstop>lineEditZDelta</tabstop>
  <tabstop>checkBoxZSync</tabstop>
  <tabstop>spinBoxCropAngle</tabstop>
  <tabstop>checkBoxShow</tabstop>
  <tabstop>sliderRed</tabstop>
  <tabstop>sliderGreen</tabstop>
  <tabstop>sliderBlue</tabstop>
  <tabstop>sliderOpacity</tabstop>
  <tabstop>pushButtonApply</tabstop>
  <tabstop>pushButtonDefault</tabstop>
  <tabstop>pushButtonDone</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
